redis/connection.py

```diff
@@ -552,7 +548,9 @@ def read_response(
             self.next_health_check = time() + self.health_check_interval
 
         if isinstance(response, ResponseError):
-            logger.exception('Parsed a response error in read_response', exc_info=response)
+            # We want to log response errors from Redis but we don't care about this one currently
+            if "SETINFO" not in response.message:
```

Comment:
Sorry to keep bouncing stuff around - we are still going to get a lot of logs here (we can get response errors for any number of reasons), can we just add this check here:

https://github.com/discord/redis-py/blob/10ad2f16d05c73e9fdc5ac1e30f6cb53c01b11cc/redis/connection.py#L397

Ultimately we want to make sure we are capturing errors there that are not the expected one (`Unknown subcommand or wrong number of arguments for 'SETINFO'`) since in the upstream version they are just being swallowed indiscriminately. Other errors I think we can rely on other logging for.